src/main/java/hudson/plugins/jacoco/JacocoPublisher.java

```diff
@@ -451,13 +461,18 @@ public void perform(@Nonnull Run<?, ?> run, @Nonnull FilePath filePath, @Nonnull
         logger.print("\n[JaCoCo plugin] Saving matched class directories for class-pattern: " + classPattern + ": ");
         for (FilePath file : matchedClassDirs) {
             dir.saveClassesFrom(file);
-            logger.print(" " + file);
```

Comment:
I have integrated jacoco plugin with a CI build job of a application having thousands of classes. The log statement prints numerous file paths on CI log which seemed unnecessary. For investigation can we add a summary message instead? Please suggest. Thanks.